books/views.py

```diff
@@ -0,0 +1,94 @@
+# Create your views here.
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+
+from models import Books, Authors, Tags, Comments
+from rest_framework import viewsets
+from serializers import BookSerializer, AuthorSerializer, TagsSerializer, CommentSerializer
+
+
+class BookList(viewsets.ModelViewSet):
+    """
+    API endpoint that allows books to be viewed or edited.
+    """
+    queryset = Books.objects.all()
+    serializer_class = BookSerializer
+
+
+class AuthorViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows authors to be viewed or edited.
+    """
+    queryset = Authors.objects.all()
+    serializer_class = AuthorSerializer
+
+
+class TagViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows tags to be viewed or edited.
+    """
+    queryset = Tags.objects.all()
+    serializer_class = TagsSerializer
+
+
+class CommentViewSet(viewsets.ViewSet):
```

Comment:
You can use `viewsets.ModelViewSet` and override `get_queryset` and `get_object` methods.